spec/fixtures/pipenv-with-lockfile/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
```

Comment:
I didn't add an example `dev` package but looking at this now I'm think I should. WDYT?